cmd/kube-proxy/app/server_others.go

```diff
@@ -79,10 +92,6 @@ func newProxyServer(
 	config *proxyconfigapi.KubeProxyConfiguration,
 	master string) (*ProxyServer, error) {
 
-	if config == nil {
```

Comment:
why remove the nil judge ?where's the double-check?

Reply:
Because if this is nil it's a bug in the program, not something that should be logged as an error for the user to deal with

Reply:
maybe it could have detected this https://github.com/kubernetes/kubernetes/pull/117690 ??